modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition.py

```diff
@@ -40,6 +42,8 @@ class PandasOnDaskDataframePartition(PandasDataframePartition):
         Call queue that needs to be executed on wrapped pandas DataFrame.
     """
 
+    execution_wrapper = DaskWrapper
```

Comment:
I would prefer to name `engine_wrapper`, which is similar to the file names we have.